OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/pointsto/package.scala

```diff
@@ -10,34 +10,32 @@ import org.opalj.br.fpcf.analyses.SimpleContextProvider
 import org.opalj.br.fpcf.properties.Context
 import org.opalj.fpcf.Entity
 import org.opalj.tac.common.DefinitionSites
-import org.opalj.value.ValueInformation
 
 package object pointsto {
 
     /**
-     * Given a definition site (value origin) in a certain method, this returns the
+     * Given a definition site PC (value origin) in a certain method, this returns the
```

Comment:
```suggestion
     * Given a definition site PC in a certain method, this returns the
```
This is not the value origin anymore

Reply:
Yes it is. Value origin is an abstraction over PCs (or TAC indices) that extends them to negative numbers for parameters and exceptions: https://github.com/opalj/opal/blob/de2be667ea96bd2b9d8abdad157e51124c473746/OPAL/ai/src/main/scala/org/opalj/ai/package.scala#L78-L81

Reply:
The double abstraction over PCs and TAC indices confused me